packages/components/src/interfaces/native/Prose.ts

```diff
@@ -6,26 +6,31 @@ import { ParagraphSchema } from "./Paragraph"
 import { TableSchema } from "./Table"
 import { UnorderedListSchema } from "./UnorderedList"
 
-export const ProseSchema = Type.Object({
-  type: Type.Literal("prose"),
-  content: Type.Optional(
-    Type.Array(
-      Type.Union([
-        DividerSchema,
-        HeadingSchema,
-        OrderedListSchema,
-        ParagraphSchema,
-        TableSchema,
-        UnorderedListSchema,
-      ]),
-      {
-        title: "Content block",
-        description: "A collection of native content components.",
-        minItems: 1,
-      },
+export const ProseSchema = Type.Object(
+  {
+    type: Type.Literal("prose", { default: "prose" }),
+    content: Type.Optional(
+      Type.Array(
+        Type.Union([
+          DividerSchema,
+          HeadingSchema,
+          OrderedListSchema,
+          ParagraphSchema,
+          TableSchema,
+          UnorderedListSchema,
+        ]),
+        {
+          minItems: 1,
+        },
+      ),
     ),
-  ),
-})
+  },
+  {
+    $id: "components-native-prose",
```

Comment:
if `$id` is not needed anymore, should this be here?

Reply:
Hmm `$id` is needed by TypeBox and JSONForms, but it is not supported by RJSF's form builder (the tool that POEs are using). The script only converts the schema from TypeBox into something that RJSF is able to understand, which they are not able to understand `$ref`s pointing to `$id`s.